content/misc/lab-setup.md

```diff
@@ -0,0 +1,34 @@
+# Setting up the Lab Environment
+
+If you have already cloned the repository, make sure it is updated:
+
+```console
+cd operating-systems
+git pull --rebase
+```
+
+The command may fail if you have uncommitted changes.
+If that is the case, commit changes and retry.
```

Comment:
```suggestion
If that is the case, stash your changes, retry, and pop the stash:

```console
student@os:~/operating-systems$ git stash

student@os:~/operating-systems$ git pull --rebase

student@os:~/operating-systems$ git stash pop
```
```

The suggestion is a bit malformed due to the backticks interfering with each other.